### ISSUE TITLE
accessibility map: allow to parameterize the time of trip and total travel time

### ISSUE THREAD

**Issue by tahini** [chairemobilite/localisation#20]
The user should be able to select something to change the parameters of the accessibility map calculations. Currently, we default to 45 minutes at 8AM

**Comment by greenscientist** [comment]
We should default to 30 minutes for the moment.

**Comment by samuel-duhaime** [comment]
@tahini Closing, we have an issue for the frontend.
